stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Messages.java

```diff
@@ -361,20 +361,6 @@ public void searchMessages() {
                 }
             });
         }
-
-        public void searchMessagesWithAttachments() {
```

Comment:
We don't have this snippet in docs: https://getstream.io/chat/docs/react/search/?language=kotlin